prediction-applier-extension/src/main/java/org/alfresco/messaging/camel/routes/HxInsightAuthClient.java

```diff
@@ -0,0 +1,41 @@
+/*-
+ * #%L
+ * Alfresco HX Insight Connector
+ * %%
+ * Copyright (C) 2023 - 2024 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.messaging.camel.routes;
+
+import org.apache.camel.CamelContext;
+import org.springframework.stereotype.Component;
+
+import org.alfresco.hxi_connector.common.adapters.auth.DefaultAuthenticationClient;
+import org.alfresco.hxi_connector.common.adapters.auth.config.properties.AuthProperties;
+
+@Component
```

Comment:
I think that annotation should not be used here since we are using XML based context.

Reply:
removed